src/v/pandaproxy/server.cc

```diff
@@ -88,6 +92,17 @@ struct handler_adaptor : ss::httpd::handler_base {
         server::reply_t rp{std::move(rep)};
         auto req_size = get_request_size(*rq.req);
         auto sem_units = co_await ss::get_units(_ctx.mem_sem, req_size);
+
+        ss::sstring access_log;
+        if (_log.is_enabled(ss::log_level::debug)) {
+            access_log = ssx::sformat(
+              R"("{} {} HTTP/{}")",
+              rq.req->_method,
+              rq.req->_url,
+              rq.req->_version);
```

Comment:
`admin_api_server` [also logs the authenticated user](https://github.com/redpanda-data/redpanda/blob/dev/src/v/redpanda/admin_server.cc#L347-L353), like this:
```
[_anonymous] GET http://cluster-1.cluster.console.svc.cluster.local.:9644/v1/node_config 
```
Should the same be done here so that we can see the same format in both here and the admin API?